cmd/simulator/config/event.go

```diff
@@ -0,0 +1,91 @@
+package config
+
+import "math/big"
+
+type EventType string
+
+const (
+	OCR3ConfigEventType     EventType = "ocr3config"
+	GenerateUpkeepEventType EventType = "generateUpkeeps"
+	LogTriggerEventType     EventType = "logTrigger"
+)
+
+type Event struct {
+	Type         EventType `json:"type"`
+	TriggerBlock *big.Int  `json:"triggerBlockNumber"`
+	Comment      string    `json:"comment,omitempty"`
+}
+
+type UpkeepType string
+
+const (
+	ConditionalUpkeepType UpkeepType = "conditional"
+	LogTriggerUpkeepType  UpkeepType = "logTrigger"
+)
+
+// OCR3ConfigEvent is an event that indicates a new OCR config should be
+// broadcast. Consult libOCR for descriptions of each config var.
+type OCR3ConfigEvent struct {
+	Event
+	// F is the configurable faulty number of nodes
+	F int `json:"maxFaultyNodes"`
```

Comment:
Could we rename this to `MaxFaultyNodes` or `MaxFaultyNodesF`?

Reply:
Changing this to be more detailed in the name helps describe it, but it also diverges from the libocr naming convention. They just call it `f`. I'm good either way. I can change it to be more in-line with the json config value for this config event though.

Reply:
I suppose ideally the config value in the JSON would be called `f`, but I think it's better to have the struct keys aligned to the JSON config keys